### PR TITLE
SB-21838 Adding opensaber api

### PR DESCRIPTION
**Pull request by sknirmalkar89**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://631.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br /> - http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — CHANGES_REQUESTED** [review]

File: apis/opensaber/opensaberapi.yaml [hunk 1]
```diff
@@ -0,0 +1,2601 @@
...
+        This api is used to create a record in registry based on the schema configuration.
+         - The endpoint for **Create Registry** is `/add`.
+         - The fields marked with an asterisk (*) are mandatory. They cannot be null or empty
+         - Schema representation for entity available such as **Teacher.json, Person.json** available by default in **public/_schema** directory under resources folder. Teacher or Person will represent the entity and other fields under Teacher or Person will represent the schema definition.
+      operationId: AddRegistryRecord
+      parameters:
+      - name: Authorization
+        in : header
+        required: true
+        description: To make use of any Group API, you require authorization. Raise a
```
[the remark this answers is not in the document]
Comment (sknirmalkar89) [reply apis/opensaber/opensaberapi.yaml:29]: Updated

File: apis/opensaber/opensaberapi.yaml [hunk 2]
```diff
@@ -0,0 +1,2601 @@
...
+                f:
+                 $ref: '#/components/examples/InternalServerErrorResponse'
+      deprecated: false
+  /read:
+    post:
+      tags:
+      - Misc
+      summary: Read Registry Record
+      description: >-
+        This api is used to read record registry
```
[the remark this answers is not in the document]
Comment (sknirmalkar89) [reply apis/opensaber/opensaberapi.yaml:74]: Done

**Comment by keshavprasadms** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://631.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br /> - http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br />  <br/> **Modified sections are -** <br />

**Comment by keshavprasadms** [comment]
Hello @sknirmalkar89,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://631.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br /> - http://631.qa.docs.sunbird.org/api-documentation/apis/opensaber// <br />  <br/> **Modified sections are -** <br />

**Review by Sunitashelar — APPROVED** [review]

**Pull request merged**